docs/DeveloperGuide.md

```diff
@@ -384,14 +384,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-**UC02: Add a debt**
+**UC02: List persons**
 
 **MSS**
 
 1.  User requests to list persons
 1.  PayMeLah shows a list of persons
-1.  User requests to add a debt to a specific person in the list
-1.  PayMeLah adds the debt to the person
 
     Use case ends.
```

Comment:
Is this for the `list` command? If so, it may be better to specifiy `list all persons`